Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs

```diff
@@ -355,6 +454,24 @@ public async Task<Dictionary<int, TrickplayInfo>> GetTrickplayResolutions(Guid i
         return trickplayResolutions;
     }
 
+    /// <inheritdoc />
+    public async Task<List<Guid>> GetTrickplayItemsAsync()
```

Comment:
```suggestion
    public async Task<IReadOnlyList<Guid>> GetTrickplayItemsAsync()
```

Also update interface